mono/utils/mono-state.c

```diff
@@ -170,6 +170,10 @@ mono_summarize_timeline_start (void)
 
 	log.level = MonoSummarySetup;
 	log.directory = configured_timeline_dir;
+	char out_file [200];
+	file_for_summary_stage (log.directory, log.level, out_file, sizeof(out_file));
+	int handle = g_open (out_file, O_WRONLY | O_CREAT, S_IWUSR | S_IRUSR | S_IRGRP | S_IROTH);
+	close(handle);
```

Comment:
Make a helper for this function and for `mono_summarize_timeline_phase_log` that just does this bit.